pkg/cmd/admin/router/router.go

```diff
@@ -105,6 +109,9 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out io.Writer)
 	cmd.Flags().StringVar(&cfg.Credentials, "credentials", "", "Path to a .kubeconfig file that will contain the credentials the router should use to contact the master.")
 	cmd.Flags().StringVar(&cfg.DefaultCertificate, "default-cert", cfg.DefaultCertificate, "Optional path to a certificate file that be used as the default certificate.  The file should contain the cert, key, and any CA certs necessary for the router to serve the certificate.")
 	cmd.Flags().StringVar(&cfg.Selector, "selector", cfg.Selector, "Selector used to filter nodes on deployment. Used to run routers on a specific set of nodes.")
+	cmd.Flags().IntVar(&cfg.StatsPort, "stats-port", 1936, "If the underlying router implementation can provide statistics this is a hint to expose it on this port.  Default value is 1936")
+	cmd.Flags().StringVar(&cfg.StatsPassword, "stats-password", cfg.StatsPassword, "If the underlying router implementation can provide statistics this is the requested password for auth.")
+	cmd.Flags().StringVar(&cfg.StatsUsername, "stats-user", "admin", "If the underlying router implementation can provide statistics this is the requested username for auth. Default value is admin")
```

Comment:
so if they set a password, we'll enable stats, and everything else (port and username) are enabled for them already by default?

what that really means is no one will get status, because they don't know they need to set `--stats-password`

if we want to enable stats by default, we could generate a random password in this command as the default value, and output it